internal/kafka/internal/workers/cluster_mgrs/cleanup_clusters_mgr_test.go

```diff
@@ -0,0 +1,203 @@
+package cluster_mgrs
+
+import (
+	"testing"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/services/sso"
+	"github.com/onsi/gomega"
+
+	apiErrors "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+var deprovisionCluster = api.Cluster{
+	Status: api.ClusterDeprovisioning,
+}
+
+func TestCleanupClustersManager_processCleanupClusters(t *testing.T) {
```

Comment:
Not to change in this PR if not wanted but I think we should start thinking on a test method naming scheme:

Sometimes I see test names written as:
Test_<TypeBeingTested>_<method_or_behavior_BeingTested>

and sometimes I see:
Test<TypeBeingTested>_<method_or_behavior_BeingTested>

Reply:
+1